acceptance-tests/src/test/resources/features/site/dataAndInformationHub.feature

```diff
@@ -51,7 +56,8 @@ Feature: Clinical Indicator hub page and sub sections
         And I should see 1 search result
 
     Scenario: CI landing pages - SHMI
-        Given I navigate to the "home" page
+        Given I navigate to the "Data and information" page
+        When I click on the accept cookie banner
```

Comment:
It would read slightly better if this `When` were replaced by `And`.

Reply:
Done. Thanks!